Backendless/Service/UserService.cs

```diff
@@ -320,7 +320,34 @@ public void AssignRole( string identity, string roleName )
                                   new object[] { identity, roleName } );
     }
 
-  #if !(NET_35 || NET_40)
+    public void ResendEmailConfirmation( String email )
+    {
+      if( email == null )
+        throw new ArgumentException( ExceptionMessage.NULL_TEMPLATE, email );
+
+      Invoker.InvokeSync<Object>( USER_MANAGER_SERVER_ALIAS, "resendEmailConfirmation", new Object[] { email } );
+    }
+
+    public void ResendEmailConfirmation( String email, AsyncCallback<Object> callback )
+    {
+      try
+      {
+        if( email == null )
+          throw new ArgumentException( ExceptionMessage.NULL_TEMPLATE, email );

```

Comment:
is NULL_TEMPLATE text appropriate for this error? (i have not checked what the error message says)